bip-nsequence-anti-fee-snipe.mediawiki

```diff
@@ -0,0 +1,116 @@
+<pre>
+  BIP: TBD
+  Layer: Applications
+  Title: Anti-fee-sniping protection with nSequence in taproot transactions to improve privacy for off-chain protocols
```

Comment:
I think the title is a bit too specific and might discourage wallets that have not or will not implement taproot to implement anti-fee-snipe at all. Maybe make the BIP a bit more general and remove "taproot" from the title? Also, the body could mention: "Any wallets that can not or do not want to implement the more complex nSequence based anti-fee-sniping may implement only nLockTime based anti-fee-snipe" (first if branch in the pseudocode)?

Reply:
The BIP only applies for taproot spends so I think it's not a good idea to remove the word taproot. It's no big deal if some wallets decide to not implement it, because the amount of cover traffic needed isn't very large, and we've got at least a decade or two until the anti-fee-sniping becomes important (if ever)